applications/matter_bridge/src/ble_providers/ble_lbs_data_provider.cpp

```diff
@@ -28,26 +28,18 @@ static bt_uuid *sUuidButton = BT_UUID_LBS_BUTTON;
 static bt_uuid *sUuidCcc = BT_UUID_GATT_CCC;
 
 #ifdef CONFIG_BRIDGE_ONOFF_LIGHT_SWITCH_BRIDGED_DEVICE
-void ProcessCommand(CommandId aCommandId, const EmberBindingTableEntry &aBinding, OperationalDeviceProxy *aDevice,
-		    void *aContext)
+void ProcessCommand(const EmberBindingTableEntry &aBinding, OperationalDeviceProxy *aDevice, Nrf::Matter::BindingHandler::BindingData &aData)
```

Comment:
I guess we don't need `Nrf` namespace here and there in this file, maybe the same for `Matter`.

Reply:
I think that Matter is required, but indeed Nrf should not be needed.